include/enclave/inside/EnclaveWasmModule.h

```diff
@@ -33,12 +37,56 @@ class EnclaveWasmModule
 
     bool loadWasm(void* wasmOpCodePtr, uint32_t wasmOpCodeSize);
 
-    bool callFunction();
+    bool callFunction(uint32_t argcIn, char** argvIn);
+
+    WASMModuleInstanceCommon* getModuleInstance();
+
+    // ---- argc/arv ----
+
+    uint32_t getArgc();
```

Comment:
The getters I need to implement in the actual modules, as the mixin does not have access to private members.